applications/nrf_desktop/src/hw_interface/Kconfig.motion

```diff
@@ -80,7 +80,7 @@ config DESKTOP_MOTION_RIGHT_KEY_ID
 	  ID of key use for generating right motion.
 
 config DESKTOP_MOTION_SIMULATED_EDGE_TIME
-	int "Time for transition between two points in a trajectory [ms]"
+	int "Time for transition between two points in a trajectory [us]"
```

Comment:
We may want to mention us resolution limitations if there are any

Reply:
Added note about precision limitation.